pokemon-app/src/styles/header.scss

```diff
@@ -0,0 +1,22 @@
+@import 'variables';
+
+header {
+    height: max-content;
+    border-bottom-left-radius: $header-footer-border-radius;
+    border-bottom-right-radius: $header-footer-border-radius;
+    background-color: $white-color;
+    box-shadow: 0px 5px 10px #5858580f;
```

Comment:
You also can create a variable for this box shadow. I see that you are using it in two places at least